CSE-Hub/user/profile.php

```diff
@@ -0,0 +1,89 @@
+<?php
+session_start();
+require_once '../sql_login/login.php';
+
+// in case the user directly want's to access this page, php doesn't know whose profile to display
+if(!isset($_GET['pass'])){
```

Comment:
Maybe giving a meaningful variable name would be more helpful (something like username , I hope you are not sending password via $_GET[ ] ).